index.html

```diff
@@ -4,9 +4,99 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+    <title>Buganda Spirit</title>
 </head>
 <body>
-    
+ 
+    <div>
+    <div class="topcontact">
+        <ul>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/facebook-f.png"/></li>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/twitter--v1.png"/></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <header>
+        <nav class="navbar">
+           <h1><a href="#"><span class="buganda">Buganda</span> <span class="bugs">KingsMen</span></a></h1>
+            <ul>
+                <li><a href="./About.html" >About</a></li>
+                <li><a href="#" >Program</a></li>
+                <li><a href="#" >Join</a></li>
+                <li><a href="#" >Sponser</a></li>
+                <li><a href="#" >New</a></li>
+                <li><button>CC Campaign</button></li>
+            </ul>
+       
+        </nav>
+</header>
+<div>
+    <div class="headding">
+      <a href="#" class="nav-h3"></a>
+      <!-- <img
+        src="https://img.icons8.com/metro/18/000000/menu.png"
+        alt="menu icon"
+      /> -->
+      <ul class='Navmenu'>
+        <li class="nav-items"><a class="nav-links" href="./About.html">About</a> </li>
+        <li  class="nav-items"><a class="nav-links"  href="#about">Program</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Join</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Sponser</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">New</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">CC Campaign</a></li>
+      </ul>
+      <div class="hamburger-menu">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+
+    </div>
+</div>
+
+<section class="intro-sections">
+            <div class="intro-wrapper">
+            <h3 class="intro-wrappers-h3">"The Buganda Pride"</h3>
+            <h1>THE BUGANDA CULTURAL PROSPEROUS FUTURE FLEX 2022</h1>
```

Comment:
Kindly, I suggest applying only 1 H1 header on your site, followed by H2, H3... in order of importance, this is good HTML practice and will help a lot with the accessibility of your site.